docs/en/sdk-manual/http-api.md

```diff
@@ -0,0 +1,215 @@
+# Clickstream HTTP API
+
+## Introduction
+
+This documentation will help you send your data directly to the Clickstream ingestion server via HTTP requests, At the
+same time, following the below specifications, your data will be correctly processed by the Clickstream data processing
+module and can be visually analyzed in the subsequent report module.
```

Comment:
--> This article provides guidance for sending clickstream data to data pipeline through HTTP request. Following below specifications, the clickstream data will be processed properly by the data pipeline.